packages/pangea-sdk/pangea/services/ai_guard.py

```diff
@@ -1,91 +1,62 @@
 from __future__ import annotations
 
-from typing import List, Optional
-
-from typing_extensions import Literal
+from typing import Any, Dict, Generic, List, Optional, TypeVar
 
 from pangea.config import PangeaConfig
-from pangea.response import PangeaResponse, PangeaResponseResult
+from pangea.response import APIResponseModel, PangeaResponse, PangeaResponseResult
 from pangea.services.base import ServiceBase
-from pangea.services.intel import UserBreachedData
-
-
-class TextGuardSecurityIssues(PangeaResponseResult):
-    compromised_email_addresses: int
-    malicious_domain_count: int
-    malicious_ip_count: int
-    malicious_url_count: int
-    redact_rule_match_count: int
-
 
-class TextGuardFindings(PangeaResponseResult):
-    artifact_count: Optional[int] = None
-    malicious_count: Optional[int] = None
-    security_issues: TextGuardSecurityIssues
 
+class AnalyzerResponse(APIResponseModel):
+    analyzer: str
```

Comment:
Missing docs?

Reply:
There are none on the website nor the OpenAPI spec.

Reply:
I see them documented on the [Dotnet SDK](https://github.com/pangeacyber/pangea-dotnet/pull/523/files#:~:text=///%20%3Csummary%3EThe%20analyzer%20name.%3C/summary%3E)